### PR TITLE
Loading of current language on setting screen was done

### PR DESCRIPTION
**Pull request by YarishPetro**
## PR Submission Reminders:
- Include a jira ticket with number and description*
- Please keep the PR as small as possible, reviewer should be able to review in less than 15 minutes
- Check if your PR havs no breaking changes

### CODE REVIEW

File: Screens/LanguageScreen.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import { setLanguage } from 'redux-i18n';
 class LanguageScreen extends Component {
     setLang = (code) => {
         this.props.storeSettings(this.props.deviceId, { languageCode: code });
-        this.props.setLanguage(code);
```
Comment (taras-tymchuk) [inline Screens/LanguageScreen.js:10]: Why did you remove this line?

Comment (YarishPetro) [reply Screens/LanguageScreen.js:10]: We don`t need it now. We call that function inside storeSettings.

**Review by vtorkalo — APPROVED** [review]
